### PR TITLE
chore: add volunteer Harsh971

### PR DESCRIPTION
**Pull request by Harsh971**

### CODE REVIEW

**Review by accodes21 — COMMENTED** [review]
Everything looks good, but can you restrict topics to less than 4.

**Comment by Harsh971** [comment]
in bio part ?

**Comment by Harsh971** [comment]
> Everything looks good, but can you restrict topics to less than 4.

In bio part ?

**Comment by Harsh971** [comment]
> Everything looks good, but can you restrict topics to less than 4.

I have made changes 🤝

File: data/users/Harsh.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Harsh Thakkar",
+  "bio": "MERN Stack, DSA",
```
Comment (codewithvoid) [inline data/users/Harsh.json:3]: Can you use title here - Like 'Web Developer'

File: data/users/Harsh.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "image": "https://avatars.githubusercontent.com/u/65072835?v=4",
+  "categories": [
+    {
+      "slug": "web-development",
+      "topics": ["JS", "ReactJs", "MySQL"]
+    },
+    {
+      "slug": "DSA and Programming",
```
Comment (codewithvoid) [inline data/users/Harsh.json:11]: use programming slug
```json
"slug": "programming"
```

**Review by codewithvoid — CHANGES_REQUESTED** [review]
Thanks for volunteering @Harsh971. Please check feedback comments.

**Comment by Harsh971** [comment]
Changes applied 👍🏻

File: data/users/Harsh971.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Harsh Thakkar",
+  "bio": "Web Developer",
+  "image": "https://avatars.githubusercontent.com/u/65072835?v=4",
+  "categories": [
+    {
+      "slug": "Web Development",
```
Comment (codewithvoid) [inline data/users/Harsh971.json:7]: use slug from [categories](https://github.com/codewithvoid/AskDevs/blob/main/public/categories.json) file

here

```json
"slug": "web-development"
```

File: data/users/Harsh971.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+      "topics": ["JS", "ReactJs", "MySQL"]
+    },
+    {
+      "slug": "Programming",
```
Comment (codewithvoid) [inline data/users/Harsh971.json:11]:
```json
"slug": "programming"
```

File: data/users/Harsh971.json [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+      "topics": ["C", "Cpp", "Java"]
```
Comment (codewithvoid) [inline data/users/Harsh971.json:12]: nit: use `c++`

File: data/users/Harsh971.json [hunk 6]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Harsh Thakkar",
+  "bio": "Web Developer",
+  "image": "https://avatars.githubusercontent.com/u/65072835?v=4",
+  "categories": [
+    {
+      "slug": "Web Development",
+      "topics": ["JS", "ReactJs", "MySQL"]
```
Comment (codewithvoid) [inline data/users/Harsh971.json:8]: Move mysql to `database-development` category

**Review by codewithvoid — CHANGES_REQUESTED** [review]

**Comment by Harsh971** [comment]
Changes Applied...

**Review by codewithvoid — APPROVED** [review]
Thanks for making the changes. Welcome to the community :)

**Pull request merged**